app/client/components/user/EditUserForm.tsx

```diff
@@ -0,0 +1,371 @@
+import * as React from "react";
+import { connect } from "react-redux";
+
+import { reduxForm, InjectedFormProps, formValueSelector } from "redux-form";
+
+import { GenderTypes } from "./types";
+import {
+  Wrapper,
+  ModuleBox,
+  ModuleHeader,
+  TopRow,
+  ButtonWrapper,
+  Row,
+  SubRow
+} from "../styledElements";
+import AsyncButton from "./../AsyncButton.jsx";
+import ProfilePicture from "../profilePicture.jsx";
+import GetVerified from "../GetVerified.jsx";
+
+import InputField from "../form/InputField";
+import SelectField from "../form/SelectField";
+import { TransferUsage } from "../../reducers/transferUsage/types";
+import { ReduxState } from "../../reducers/rootReducer";
+import { TransferAccountTypes } from "../transferAccount/types";
+import QrReadingModal from "../qrReadingModal.jsx";
+import { replaceUnderscores } from "../../utils";
+
+export interface IEditUser {
+  firstName?: string;
+  lastName?: string;
+  publicSerialNumber?: string;
+  phone?: string;
+  bio?: string;
+  referredBy?: string;
+  location?: string;
+  businessUsage?: string;
+  usageOtherSpecific?: string;
+  oneTimeCode: number;
+  failedPinAttempts: number;
+  accountType: any[TransferAccountTypes];
+  [key: string]: any;
```

Comment:
Is there a way we can re-write this to avoid needing to allow basically any key with any value? Feels like it diminished the utility of typescript significantly.